pages/report/index.tsx

```diff
@@ -15,6 +16,7 @@ const Report: NextPage = () => {
                     Reports
                 </Heading>
             )}
+            <ReportTable entries={reportsRespose} />
```

Comment:
It would be a good idea to render conditionally here i.e.
```typescript
{reportsRespose.isSuccess && <ReportTable entries={reportsRespose.data} />}
```
That way you won't have to worry about `isSuccess` in `ReportTable` and it improves readibility.

Reply:
And maybe rename `reportsRespose` -> `reportsResponse`